scripts/send-sms.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const config = require('../config').getProperties()
+const NOT_SET = 'YOU MUST CHANGE ME'
+
+if (config.sms.apiKey === NOT_SET || config.sms.apiSecret === NOT_SET) {
+  fail('This isn\'t going to work.')
```

Comment:
`This isn\'t going to work.` 😁, mind appending `Please check configuration.` or something similar to end of message?